Sources/EmbraceConfigInternal/EmbraceConfigurable/EmbraceConfigurable.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Copyright © 2024 Embrace Mobile, Inc. All rights reserved.
+//
+
+import Foundation
+
+/// This protocol is used to add configuration to the runtime of the SDK
+/// It is used to configure the ongoing behavior of the SDK
+@objc public protocol EmbraceConfigurable {
```

Comment:
We'll need to move this protocol into a target that is meant for public consumption. It will also need to be a dependency of EmbraceConfigInternal. Will need to do this so clients don't need to `import EmbraceConfigInternal`